safe_web_api_playground/README.md

```diff
@@ -10,7 +10,9 @@ Instead of having to upload your site to the network every time you want to test
 - clone this repository and `cd` into the respective directory
 - Run `yarn`
 - Run `yarn gulp` to start server
-- Visit `localhost:3003` in your SAFE browser to see page
+- Visit `localhost://p:3003` in your mock SAFE browser to see page
+- Be sure that `File > SAFE Browsing Enabled` is unchecked
+- Note: this will soon be changing to `http://localhost:3003` with the new browser.
```

Comment:
@hunterlester , I would change this and firmly state which are the two formats for each browser.